client/wfm/src/workorder/WorkOrder.ts

```diff
@@ -1,14 +1,19 @@
 import * as Promise from 'bluebird';
 import { min } from 'lodash';
-import { getRoundedStatus, Task, TaskStatus } from '../task/Task';
+
+type LatLng = [number, number];
+
 /**
  * The executable instance of a {@link Process}
  */
-export interface ProcessInstance {
+export interface WorkOrder {
   /**
    * Unique identifier for this ProcessInstance
    */
   id: string;
+
+  type: string;
```

Comment:
Let's document this. Cannot find for what we use this type.  It looks like it's used only for label so `label` or name will be more appropriate:
https://github.com/feedhenry-raincatcher/raincatcher-angularjs/search?utf8=%E2%9C%93&q=workorder.type&type=